zerver/data_import/hipchat.py

```diff
@@ -89,6 +89,13 @@ def process(in_dict: ZerverFieldsT) -> ZerverFieldsT:
         short_name = in_dict['mention_name']
         timezone = in_dict['timezone']
 
+        if not email:
+            # Hipchat guest users don't have emails, so
+            # we just fake them.
+            assert(is_guest)
+            email = 'guest-{id}@example.com'.format(id=id)
```

Comment:
Do we know it's just guest users missing these?  If not, we could do `user-{id}` instead.

Reply:
I assert `is_guest` above it.  If that assertion holds, which it makes sense it would, I think `guest-` is a better prefix for the emails.